brainstormforce/astra
# AST-1496 - Hiding before & after properties which was adding unnecessary oval shape when color control option was selected/clicked

### Description
Hiding before & after properties which were adding unnecessary oval shape when color control option was selected/clicked.

### Screenshots

Note: Customizer Background color which is visible in the screencast is added just for testing colors options correctly.

Before Fixing Oval shape issue - https://share.getcloudapp.com/8Luprxjl
After Fixing Oval shape issue  - https://share.getcloudapp.com/yAuklxlW

### Types of changes
Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?
1. Open header or footer builder in customizer.
2. Add widget.
3. Add any text or heading block.
4. Open `Show more settings` for that block.
5. In color picker controller, try selecting/clicking any color option.
6. After selecting, you will find an oval shape is displayed in the color option.

### Checklist:
